commands/address.go

```diff
@@ -142,7 +143,17 @@ var walletImportCmd = &cmds.Command{
 		cmdkit.FileArg("walletFile", true, false, "File containing wallet data to import").EnableStdin(),
 	},
 	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
-		addrs, err := GetAPI(env).Address().Import(req.Context, req.Files)
+		iter := req.Files.Entries()
+		if !iter.Next() {
+			return fmt.Errorf("no file given: %s", iter.Err())
+		}
+
+		fi, ok := iter.Node().(files.File)
+		if !ok {
+			return fmt.Errorf("given file was not a files.File")
```

Comment:
Since this message will go to the user, perhaps avoid Go terms and say "%s is not a file"

Reply:
Same issue as stated here: https://github.com/filecoin-project/go-filecoin/pull/2100#discussion_r260047378